catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/EnumerationValidator.java

```diff
@@ -95,13 +106,18 @@ public boolean equals(Object o) {
 
         EnumerationValidator that = (EnumerationValidator) o;
 
+        if (ignoreCase != that.ignoreCase) {
```

Comment:
You can `append()` these to the EqualsBuilder: `.append(ignoreCase, that.ignoreCase)`